components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/model/mgwdefinition/MgwAPIDefinition.java

```diff
@@ -36,6 +36,8 @@
     private MgwPathsDefinition pathsDefinition;
     private String security; //todo: bring enum
     private APICorsConfigurationDTO corsConfiguration;
+    //to identify whether it has been used
+    private boolean isUsed = false;
```

Comment:
If we rename like this we can remove the comment
```suggestion
    private boolean isDefinitionUsed = false;
```